client/src/app/shared/components/choice-dialog/choice-dialog.component.html

```diff
@@ -0,0 +1,11 @@
+<h2 mat-dialog-title>{{ data.title | translate }}</h2>
+
+<mat-radio-group class="choice-radio-group" [(ngModel)]="selectedChoice">
```

Comment:
that would "just" enable one single choice. That might be extended (later) using `@input` variables to also support multiple choice operations.